app/scripts/templates/partial/coppa-age-input.mustache

```diff
@@ -0,0 +1,4 @@
+<div class="input-row">
+  <label class="label-helper"></label>
+  <input type="number" id="age" placeholder="{{#t}}How old are you?{{/t}}" min="0" max="150" value="{{ age }}" {{#shouldFocusAge}}autofocus{{/shouldFocusAge}} required />
```

Comment:
`max` seems to be ignored.

![ignore-max-age](https://cloud.githubusercontent.com/assets/848085/9353032/59764e8e-465d-11e5-835a-dbac0e0e541d.gif)